libraries/AP_ToshibaCAN/AP_ToshibaCAN.cpp

```diff
@@ -372,8 +428,24 @@ void AP_ToshibaCAN::update()
         uint64_t time_us = AP_HAL::micros64();
         for (uint8_t i = 0; i < MIN(TOSHIBACAN_MAX_NUM_ESCS, 8); i++) {
             if (_telemetry[i].new_data) {
-                df->Write_ESC(i, time_us, _telemetry[i].rpm * 100U, _telemetry[i].millivolts * 0.1f, 0, 0, 0);
+                df->Write_ESC(i, time_us, _telemetry[i].rpm * 100U, _telemetry[i].millivolts * 0.1f, 0, _telemetry[i].temperature, 0);
                 _telemetry[i].new_data = false;
+                /*gcs().send_text(MAV_SEVERITY_CRITICAL,"ut:%ld vt:%ld wt:%ld mt:%ld 0:%x 1:%x 2:%x 3:%x 4:%x",
+                        (unsigned long)u_temp,
+                        (unsigned long)v_temp,
+                        (unsigned long)w_temp,
+                        (unsigned long)motor_temp,
+                        (unsigned int)_last_reply_data2.data[0],
+                        (unsigned int)_last_reply_data2.data[1],
+                        (unsigned int)_last_reply_data2.data[2],
+                        (unsigned int)_last_reply_data2.data[3],
+                        (unsigned int)_last_reply_data2.data[4]
+                        );*/
+                /*gcs().send_text(MAV_SEVERITY_CRITICAL,"ut:%ld vt:%ld wt:%ld",
+                                        (unsigned long)u_temp,
+                                        (unsigned long)v_temp,
+                                        (unsigned long)w_temp
+                                        );*/
```

Comment:
Left-over?

Reply:
yes, debug removed in follow-up commit. txs